File: cs50/pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;

    do
    {
        height = get_int("Enter positive height:");
    }
    while (height < 0 || height > 23);

    for (int i = 0; i < height; i++)
    {
        for (int j = 0, first_hash_index = height - i - 1; j <= height; j++)
        {
            if (j < first_hash_index)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: cs50/pset6/mario/mario.py
from cs50 import get_int

# Prompt the user for the height of the pyramid.
while True:
    n = get_int("Enter height: ")
    # Make sure the input integer is within the allowed range.
    if n >= 0 and n <= 23:
        break

# Print the pyramid.
for i in range(1, n + 1):
    # print spaces
    print(" " * (n - i), end="")
    # print #'s
    print("#" * (i + 1), end="")
    # print new line
    print()